engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -750,7 +750,7 @@ void Canvas::DrawArc(const Arc& arc, const Paint& paint) {
     //
     // TODO(flar): It also only supports Butt and Square caps for now.
     // See https://github.com/flutter/flutter/issues/169400
-    if (oval_bounds.IsSquare() && paint.stroke.cap != Cap::kRound) {
+    if (oval_bounds.IsSquare()) {
```

Comment:
You can delete the last 2 lines of the preceding comment (the todo line with the link to the issue) since you are fixing that bug.